src/views/GovernanceReviewTeam/Dates/index.tsx

```diff
@@ -177,10 +180,9 @@ const Dates = ({ systemIntake }: { systemIntake: SystemIntake }) => {
                           {t('general:date.year')}
                         </Label>
                         <Field
-                          as={TextField}
+                          as={DateInputYear}
```

Comment:
The year field in the GRT/GRB dates doesn't enforce four digits like it does in the Issue LCID page

Reply:
The responsibility of this PR is to add help text to date fields, not enforce validations. Does this change cause a regression? If not, I don't think it's in scope.

Reply:
Thats fair, I was mainly commenting b/c it seemed like based on the code (the minLength field on the DateInput) that the help text should be displayed in that case. I'm guessing that the help text is somehow tied to the validation schema?